engines/ultima/nuvie/actors/actor_manager.cpp

```diff
@@ -1087,19 +1083,22 @@ bool ActorManager::loadCustomTiles(nuvie_game_t game_type) {
 
 	tile_manager->freeCustomTiles(); //FIXME this might need to change if we start using custom tiles outside of ActorManager. eg custom map/object tilesets
 
-	loadCustomBaseTiles(datadir);
-	loadAvatarTiles(datadir);
-	loadNPCTiles(datadir);
+	loadCustomBaseTiles();
+	if (obj_manager->use_custom_actor_tiles()) {
+		loadAvatarTiles(datadir);
+		loadNPCTiles(datadir);
+	}
 
 	return true;
 }
 
-void ActorManager::loadCustomBaseTiles(const Common::Path &datadir) {
+void ActorManager::loadCustomBaseTiles() {
+	Common::Path datadir = "mods";
 	Common::Path imagefile;
 	build_path(datadir, "custom_tiles.bmp", imagefile);
 
 	//attempt to load custom base tiles if the file exists.
-	tile_manager->loadCustomTiles(Game::get_game()->get_data_file_path(imagefile), true, true, 0);
+	tile_manager->loadCustomTiles(imagefile, true, true, 0);
```

Comment:
Could you perform the same changes in `ActorManager::loadAvatarTiles()`?

Reply:
yeah I left the avatar tiles as is mainly because they are already included in the `ultima.dat` file that ships with ScummVM. I think going forward I might change the archive logic to allow overriding individual files from the `ultima.dat` file using the extraPathDir. Currently if you set the extra path it overrides all files form the archive. But I might do that in another PR.

Reply:
In this case, it should be possible to load custom assets both from `ultima.dat` and external folders (with external folders having higher priority), regardless of their type

Reply:
yeah good point. I'll just make that change now instead of having a special case for this type of mod.